src/App.jsx

```diff
@@ -30,8 +31,8 @@ export default function App() {
   const [accountLoaded, setAccountLoaded] = useState(false);
   const [accountAddress, setAccountAddress] = useState("");
 
-  //const WS_PROVIDER = "ws://127.0.0.1:9944";
-  const WS_PROVIDER = "wss://dev-node.substrate.dev:9944";
+  const WS_PROVIDER = "ws://127.0.0.1:9944";
+  //const WS_PROVIDER = "wss://dev-node.substrate.dev:9944";
```

Comment:
Until we have a network selector, please keep the default node the `dev-node`, thus when we deploy it will always connect to something.

Reply:
Oops, my bad.